bazel/cas/service.go

```diff
@@ -8,10 +8,10 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"math/rand"
 	"net"
 	"sync"
 	"time"
-
```

Comment:
Could you keep this newline? we like to keep our imports separated by stdlib / 3rdparty / scoot stuff